parser/parser/cc/grammars/typedruby.ypp

```diff
@@ -863,10 +863,14 @@ int yylex(parser::semantic_type *lval, ruby_parser::location *lloc, ruby_parser:
 
       expr_value: expr
 
-   expr_value_do:   {
+   expr_value_do_begin:
```

Comment:
WDYT about calling this `expr_value_do_push_cond` or somesuch?  Partly so it's a little easier to see that it needs to match up with `driver.lex.cond.pop()` and partly because suffixing with "begin" makes me think this is actually about `begin`, which is not true.

Reply:
irrelevant, as i factored it another way (though thanks for the suggestion! i would have taken it had i not found a different way).